app/importers/concerns/versionable_resource.rb

```diff
@@ -18,6 +18,7 @@ def import
         update_metadata(available_version)
         Rails.logger.info "#{self.class.name}: resource updated, new data indexed."
       else
+        super if self.class.const_defined?(:CONTAINS_MAPPER_LOOKUPS) # Run the importer anyways to catch Mapper updates
```

Comment:
Instead of adding this trailing conditional, which confuses the if/else block somewhat, wouldn't it be better to adjust the first if? i.e.:
```
if resource_changed? || self.class.const_defined?(:CONTAINS_MAPPER_LOOKUPS)
```
You can interpret this to mean "do the import if the source document has changed or we need to do mapper lookups (which we cannot know have changed or not)". It's more representative of our thinking here therefore communicates the intent more clearly.